clinical-validation.wdl

```diff
@@ -25,17 +25,19 @@ import "tasks/rtg.wdl" as rtg
 import "tasks/samtools.wdl" as samtools
 import "tasks/vt.wdl" as vt
 
+struct ValidationUnit {
+    File callVcf
+    File baselineVcf
+    String outputPrefix
```

Comment:
Why not drop this outputPrefix and use the `basename(callVcf)` as prefix?

Reply:
That was done to allow users to specify multiple VCF files that have the same name. That way, if you run your pipeline with 5 settings, you do not need to rename the output VCF files to use them in this pipeline to avoid collisions.

Reply:
Sounds good. It could also be made optional `String? outputPrefix` and be used with `select_first([unit.outputPrefix, basename(callVcf)])`

Reply:
I think in practice it is not really optional, since if you do not specify it you might get collisions if you use a cromwell output folder. So it would be a bit sneaky to make it an optional input and then have the workflow crash (potentially, but still) when people leave it out.